### PR TITLE
related to #8978: fixing non-GUI-parts (img delete, remove exif, move img fct to helper)

### PR DESCRIPTION
**Pull request by eddiemuc**
Contains the NON-GUI-parts for #8978 :
* moves image select functionality in separate helper class `ImageActivityHelper` so it can be used elsewhere too
* ensures that EXIF information in images is ALWAYS removed
* adds delete-logic to ensure that no non-used image data is kept on users device
* some minor fixes (load image thumbnals asynchronously, ensure image default caption is correct on image deletion, center image view in image select activity)

### CODE REVIEW

File: main/src/cgeo/geocaching/ImageSelectActivity.java [hunk 1]
```diff
@@ -196,129 +196,38 @@ private void syncEditTexts() {
...
+            loadImagePreview();
+        });
     }
 
     private void selectImageFromStorage() {
-        final Intent intent = new Intent(Intent.ACTION_GET_CONTENT);
-        intent.setType("image/*");
 
-        startActivityForResult(Intent.createChooser(intent, "Select Image"), SELECT_STORED_IMAGE);
+        //TODO: revert to single
```
Comment (moving-bits) [inline main/src/cgeo/geocaching/ImageSelectActivity.java:209]: something yet to be done, or an outdated comment?

Comment (eddiemuc) [reply main/src/cgeo/geocaching/ImageSelectActivity.java:209]: Danm, I forgot to remove the TODO tag. But the change itself was done. I can remove the false comment tomorrow (don't have access to source right now)

Comment (eddiemuc) [reply main/src/cgeo/geocaching/ImageSelectActivity.java]: > something yet to be done, or an outdated comment?

Todo is now removed

Comment (moving-bits) [reply main/src/cgeo/geocaching/ImageSelectActivity.java]: thanks

**Pull request merged**